### DISCUSSION TITLE
ADM Validation Tool

### DISCUSSION THREAD

**Discussion by gurumurthy-godlaveeti** [atlasmap/atlasmap#4213] [Design & Features]
Hi Team,

Do we have any tool to validate the ADM by providing the data.

As a scenario, we prepared the ADM template to transform json-to-json. We were able to test ADM by passing source json to see target json (transformed) is coming proper or not.

But We wish to check if any existed tool already taking ADM + Source JSON to get Target JSON in UI based for verification.

Thanks & Regards,
Gurumurthy

**Comment by igarashitm** [comment]
That sounds nice. I'm not aware of any existing tool like that. We have an issue for the RFE to introduce whole mappings preview (#452) but at this moment noone plans to work on it shortly.
